staubli_resources/urdf/common_colours.xacro

```diff
@@ -2,9 +2,15 @@
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
 
   <!-- RAL 1028 -->
-  <xacro:property name="colour_staubli_yellow" value="1.0 0.608 0 1" />
+  <xacro:property name="melon_yellow" value="1.0 0.608 0.0 1.0" />
+
+  <!-- RAL 1018 -->
+  <xacro:property name="zinc_yellow" value="0.973 0.953 0.208 1.0" />
 
   <!-- RAL 9016 -->
-  <xacro:property name="colour_staubli_white" value="0.945 0.941 0.918 1" />
+  <xacro:property name="traffic_white" value="0.965 0.965 0.965 1.0" />
+	
+  <!-- RAL 7043 -->
+  <xacro:property name="traffic_grey_b" value="0.306 0.329 0.322 1.0" />
 
 </robot>
```

Comment:
General comment: I've asked Staubli directly, and RAL1028 is the official 'staubli yellow'. It would seem RAL1018 is not. It is however perfectly possible to get a robot in RAL1018, as you can get them in just about any colour.

I have some more info on other colours and will add them once we get this merged.